app/controllers/bookings_controller.rb

```diff
@@ -25,31 +27,46 @@ def create
 
   def show
     @booking = Booking.find(params[:id])
+    authorize @booking
   end
 
 
   def accept
     @booking = Booking.find(params[:id])
     @booking.status = "Confirmed"
     @booking.save
-      redirect_to activity_booking_path([@booking.activity, @booking])
+    redirect_to activity_booking_path([@booking.activity, @booking])
+    skip_policy_scope
+    skip_authorization
```

Comment:
@Fran-MO I used these 2 lines so that pundit does not authorize the method for a specific user, I'm unsure what the scope of this method is but we can authorize it after the method is completed.
I used the same for the Decline method too.